config.yml

```diff
@@ -36,3 +36,5 @@ outputs:
   home:
     - html
     - json
+
+buildFuture: true
```

Comment:
Not sure why this is included in this PR, though I think it's fairly harmless at the moment.

Reply:
Its so we can set the real date of an event in the future, and it be rendered in testing / preview